README.md

```diff
@@ -0,0 +1,73 @@
+[![image](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

Comment:
Do you actually use `black`? I am not familiar with the usage of `black`, but it may be possible to integrate it as pre commit hook or used somehow inside the CI to ensure code contributions are indeed formatted with `black`. This is maybe worth an extra issue. But for now, lets keep the badge...

Reply:
I use `black` to auto-format my code. I've no particular affinity to it other than it seems a convenient tool for this purpose. I'm happy to follow your lead regarding whether we should specify some kind of consistent formatting, for example with `black` or with something else.